policyengine_us/parameters/gov/states/vt/tax/income/credits/charitable/cap.yaml

```diff
@@ -0,0 +1,15 @@
+description: Vermont caps the charitable contributions credit at this amount.
```

Comment:
unify this with rate.yaml as a scale parameter. note it caps *contributions* at this amount, not the credit